sites/public/pages/index.tsx

```diff
@@ -57,7 +57,7 @@ export default function Home(props: IndexProps) {
       <Hero
         title={heroTitle}
         buttonTitle={t("welcome.seeRentalListings")}
-        buttonLink="/listings"
+        buttonLink="/listings?page=1"
```

Comment:
Do we still need the page param if it assumes no param is 1?

Reply:
This avoids that weird lag we were talking about when the param isn't provided. Haven't figured out the deal with that yet, but this seems ok to do unitl then.